server/boot/index.js

```diff
@@ -51,7 +49,16 @@ function setup() {
 	app.use( '/calypso', express.static( path.resolve( __dirname, '..', '..', 'public' ) ) );
 
 	// service-worker needs to be served from root to avoid scope issues
-	app.use( '/service-worker.js', express.static( path.resolve( __dirname, '..', '..', 'client', 'lib', 'service-worker', 'service-worker.js' ) ) );
+	app.use( '/service-worker.js',
+		express.static( path.resolve( __dirname, '..', '..', 'client', 'lib', 'service-worker', 'service-worker.js' ) ) );
+
+	// loaded when we detect stats blockers - see lib/analytics/index.js
+	app.get( '/nostats.js', function( request, response ) {
+		const analytics = require( '../lib/analytics' );
+		analytics.tracks.recordEvent( 'calypso_nostats', {}, request );
+		response.setHeader( 'content-type', 'application/javascript' );
+		response.end( "console.log('Your browser appears to be blocking our stats');" );
```

Comment:
We probably don't want to return a `console.log` ... some older browsers will fatally error. We can still do it, just put a guard around it to make sure `console` and `console.log` exist.

Reply:
Actually, it might be a good idea to do a `console.warn` and tell them they're making it harder to debug issues or something. Guilt trip them :p

Reply:
Browsers that don't support console.log won't run Calypso anyway. I'm not worried about them.

https://developer.mozilla.org/en-US/docs/Web/API/Console/log#Browser_compatibility

That said, it might not make sense to have any output at all. For now we could just return an empty string.